app/src/common/docs/DocExample.module.scss

```diff
@@ -31,28 +31,30 @@
 }
 
 .preview-container {
-    background: $white;
-    border: 1px solid $gray40;
+    background: var(--uui-docs-background);
+    border: 1px solid var(--uui-secondary-20);
     width: 960px;
     max-width: 100%;
+
 }
 
 .code {
     font-size: 14px;
     line-height: 18px;
     padding: 24px;
     margin: 0;
-    border-top: 1px solid $gray40;
-    background-color: white;
+    border-top: 1px solid var(--uui-secondary-20);
+    background-color: var(--uui-surface);
+    color: var(--uui-text-primary);
```

Comment:
There is no need to set the heading color here, it is already set in the typography